esp_idf/spotflow/srcs/net/spotflow_mqtt.c

```diff
@@ -0,0 +1,94 @@
+
+#include "net/spotflow_mqtt.h"
+
+static const char *TAG = "spotflow_mqtt";
+
+esp_mqtt_client_handle_t client = NULL;
+bool mqtt_connected = false;
+
+#if CONFIG_BROKER_CERTIFICATE_OVERRIDDEN == 1
+static const uint8_t mqtt_spotflow_io_pem_start[]  = "-----BEGIN CERTIFICATE-----\n" CONFIG_BROKER_CERTIFICATE_OVERRIDE "\n-----END CERTIFICATE-----";
+#else
+extern const uint8_t mqtt_spotflow_io_pem_start[]   asm("_binary_mqtt_spotflow_io_pem_start");
+#endif
+extern const uint8_t mqtt_spotflow_io_pem_end[]   asm("_binary_mqtt_spotflow_io_pem_end");
```

Comment:
`mqtt_spotflow_io_pem_end` is unused (and incorrect if `CONFIG_BROKER_CERTIFICATE_OVERRIDDEN == 1`).